### DISCUSSION TITLE
history gem allure-cucumber

### DISCUSSION THREAD

**Discussion by evertonalmeida33** [allure-framework/allure2#1920] [General Discussion]
Como utilizar o histórico de execuções com a gem allure-cucumber?

**Comment by cheshi-mantu** [comment]
> How do I use execution history with gem allure-cucumber?

Not sure I understood the question, please check [here](https://docs.qameta.io/allure-report/#_history_files) to know about history trends in the report.

Allure TestOps supports the history over all the time you are executing the tests out of the box.
